content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -407,22 +442,39 @@ Then create the check using sensuctl and the `disk_usage-check.json` file includ
 sensuctl create --file disk_usage-check.json
 {{< /highlight >}}
 
-We should see it working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
+You don't need to make any changes to disk_usage-check.json before running `sensuctl create --file disk_usage-check.json`.
+
+You should see the check working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
 
 {{< highlight shell >}}
 sensuctl event list
 {{< /highlight >}}
 
-Now we should be able to see disk usage metrics for the sandbox in [Grafana](http://localhost:4002/d/go02/sensu-go-sandbox-combined).
+Now, you should be able to see disk usage metrics for the sandbox in Grafana: [reload your Grafana tab to show the Sensu Go Sandbox Combined](http://localhost:4002/d/go02/sensu-go-sandbox-combined).
 
 You made it! You're ready for the next level of Sensu-ing.
-Here are some resources to help continue your journey:
 
-- [Install Sensu Go](../../installation/install-sensu)
-- [Collect StatsD metrics](../../guides/aggregate-metrics-statsd)
-- [Create a read-only user](../../guides/create-read-only-user/)
+Before you move on, take a moment to remove the virtual machine and resources installed during this sandbox lesson. Run:
+{{< highlight shell >}}
+vagrant destroy
+{{< /highlight >}}
```

Comment:
Now when I run `vagrant destroy` at this point, I get:
```
[sensu_go_sandbox]$ vagrant destroy
-bash: vagrant: command not found
```

I have gone through the lesson twice with this result (after Aaron helped me ID the previous issue). I deleted the VM manually via Virtual Box interface before going back through the lesson the second time.

Reply:
Command prompt here indicates you are still inside the `sensu_go_sandbox` VM, so you need to use ctrl-D again to exit that ssh session before the `vagrant` command is available again in the context of the VM host system (your laptop).

Reply:
Yes, that was it! Thank you -- I had a feeling I had been looking at this for too long.